genienlp/server.py

```diff
@@ -168,6 +188,10 @@ def parse_argv(parser):
     parser.add_argument('--stdin', action='store_true', help='Interact on stdin/stdout instead of TCP')
     parser.add_argument('--locale', default='en', help='locale tag of the language to parse')
 
+    # for confidence estimation:
+    parser.add_argument('--calibrator_path', type=str, default=None, help='If provided, will be used to output confidence scores for each prediction.')
```

Comment:
This should default to `--path` + /calibrator.pt or similar, and then you check if the file exists or not.

Reply:
Fixed this.